src/windows/CameraProxy.js

```diff
@@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
             if (sensor !== null) {
                 sensor.addEventListener("orientationchanged", onOrientationChange);
             }
-            capturePreview.addEventListener('click', captureAction);
+            //capturePreview.addEventListener('click', captureAction);
```

Comment:
you can iterate with a new commit, I can merge them. Also before I can merge your changes, you need to sign Apache's Contributor License Agreement (can be done online): http://www.apache.org/licenses/#clas